utils/defillama.ts

```diff
@@ -7,19 +7,37 @@ interface TvlData {
   totalLiquidityUSD: number;
 }
 
-export async function getDefillamaTvl() {
-  const { data } = await axios.get(baseUrl);
+export async function getDefillamaTvl(url: string = baseUrl) {
+  const { data } = await axios.get(url);
   const defillamaTvlData: TvlData[] = data.tvl.slice(-1);
   const latestTvl = defillamaTvlData[0].totalLiquidityUSD;
   return latestTvl;
 }
 
-export async function getDefillamaPercentChange() {
-  const { data } = await axios.get(baseUrl);
+export async function getDefillamaPercentChange(url: string = baseUrl) {
```

Comment:
This function was easy to re-use. Just replace our DefiLlama api url with whatever value comes from Contentful